src/github/issue.ts

```diff
@@ -36,3 +45,6 @@ export default async (context: CustomContext, _: JiraClient, util): Promise<void
 		githubResponse?.status
 	);
 };
+
+const updateIssue = async (githubClient:GitHubAPI | GitHubClient, issue: Octokit.IssuesUpdateParams) =>
```

Comment:
I do the same call in the PR event, is it worth me calling this one or having a service for both events(pr and issue) to access?

Reply:
always share code where we can.  I'd say move it to it's own file somewhere with a TODO once the flags have been removed.